docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -483,6 +483,13 @@ extra operands ``i32 1`` and ``i32 2``.
 decorates the argument ``b`` of ``k`` with ``Restrict`` in SPIR-V while not
 adding any decoration to argument ``a``.
 
+Loop controls and loop metadata
+-------------------------------
+
+SPIR-V Loop controls that do not have a corresponding community LLVM metadata
```

Comment:
Should we specify where the metadata should be applied eg latch basic block?

Reply:
Updated